src/Controls/src/Core/HybridWebView/HybridWebView.cs

```diff
@@ -173,7 +97,6 @@ private void AsyncTaskCompleted(string taskId, string result)
 		/// <param name="paramValues">Optional array of objects to be passed to the JavaScript method by JSON-encoding each one.</param>
 		/// <param name="paramJsonTypeInfos">Optional array of metadata about serializing the types of the parameters specified by <paramref name="paramValues"/>.</param>
 		/// <returns>An object of type <typeparamref name="TReturnType"/> containing the return value of the called method.</returns>
-#if WINDOWS || ANDROID || IOS || MACCATALYST
 		public async Task<TReturnType?> InvokeJavaScriptAsync<TReturnType>(string methodName, JsonTypeInfo<TReturnType?> returnTypeJsonTypeInfo, object?[]? paramValues = null, JsonTypeInfo?[]? paramJsonTypeInfos = null)
```

Comment:
Maybe this should be the API definition for the `IHybridWebView` interface since it has the most data and is the most correct.

Also, maybe this should call into the handler so the handler can also deserialize the return type. This will keep all the logic inside the handler for people to override.

Reply:
To me it's a bit of a philosophical question. Overriding how methods are called - sure, I guess I can see that, especially for adding a new platform. But a platform shouldn't need to participate in how the JSON stuff is done.

Reply:
This is cleaned up now because the `string` one is gone completely. Only strongly-typed remains, so it's the only invoke method on the interface and also of course the implementation.